nginx_proxy/start_nginx.sh

```diff
@@ -42,4 +79,11 @@ if [[ -f "${CONF_FILE}" ]]; then
   cp "${CONF_FILE}" /etc/nginx/nginx.conf
 fi
 
+# fetch Service Configuration from Service Management if the service name and
+# service version are provided.
+if [[ -n "${SERVICE_NAME}" && -n "${SERVICE_VERSION}" ]]; then
+  /usr/sbin/fetch_service_config.sh \
+    -s "${SERVICE_NAME}" -v "${SERVICE_VERSION}" || exit $?
+fi
```

Comment:
Maybe print an error message and return non-zero if only one of -n or -v is provided?